addons/web_editor/static/src/xml/editor.xml

```diff
@@ -98,6 +98,27 @@
                         <li><a class="dropdown-item" href="#">62</a></li>
                     </ul>
                 </div>
+                <div t-if="props.showFontSize" id="font-size" class="btn-group dropdown">
+                    <button type="button" class="btn dropdown-toggle"
+                        data-bs-toggle="dropdown" tabindex="-1"
+                        data-bs-original-title="Font Size" aria-expanded="false">
+                        <span id="fontSizeCurrentValue" title="Font size"></span>
+                    </button>
+                    <ul class="dropdown-menu">
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize">default</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="8px">8</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="9px">9</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="10px">10</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="11px">11</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="12px">12</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="14px">14</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="18px">18</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="24px">24</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="36px">36</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="48px">48</a></li>
+                        <li><a class="dropdown-item" href="#" data-call="setFontSize" data-arg1="62px">62</a></li>
+                    </ul>
+                </div>
```

Comment:
Ouch, we definitely don't want to duplicate that list. We just want the input to not react like an input if the option is not set.

Reply:
OK, let me make it that way. Thanks!

Reply:
@dmo-odoo i made the changes. Have a look Thanks!